web-ui/src/main/resources/catalog/components/viewer/wmsimport/partials/wmsimport.html

```diff
@@ -31,10 +31,12 @@
         <div class="input-group">
           <input class="form-control" type="text" data-toggle="dropdown"
                  data-ng-model="url" data-ng-change="load()" data-ng-model-options="{debounce:500}"
+                 aria-label="{{'serviceUrl' | translate}}"
```

Comment:
do you really want to set the actual URL as `aria-label` instead of some label like _Service URL_ or something similar?